File: tests/fixtures/multi_lang_repo/module.py
"""Python file in multi-language repository."""


def python_function(x: int) -> int:
    """Simple Python function.

    Args:
        x: Input value

    Returns:
        Doubled value
    """
    return x * 2


class PythonClass:
    """Simple Python class."""

    def __init__(self, value: int):
        """Initialize with value."""
        self.value = value

    def get_value(self) -> int:
        """Get the value."""
        return self.value

File: tests/fixtures/multi_lang_repo/module.cpp
// C++ file in multi-language repository

#include <string>

namespace Multi {

// Simple function
int cppFunction(int x) {
    return x * 2;
}

// C++ class
class CPPClass {
private:
    int value;

public:
    CPPClass(int v) : value(v) {}
    
    int getValue() const {
        return value;
    }
};

} // namespace Multi
